packages/i18n/src/test/index.js

```diff
@@ -85,3 +89,20 @@ describe( 'i18n', () => {
 		} );
 	} );
 } );
+
+//test setting additional localeData
+setLocaleData( additionalLocaleData, 'test_domain' );
+
+describe( 'i18n' , () => {
```

Comment:
I guess it's better if `describes` are unique the `i18n` is already defined above, I'd rename this one `setLocaleData` and move it inside the previous `i18n` describe. And probably wrap `setLocaleData( additionalLocaleData, 'test_domain' );` in a `beforeAll` inside the `setLocaleData` describe. 

Not sure that sentence is clear :P

Reply:
no the sentence wasn't entirely clear, but after a little googling I _think_ I got the gist of what you wanted?